## This repository contains online material from the AfricanNeo Project (Fortes-Lima et al. ***Nature*** 2023).
###
#### - Citation
***The genetic legacy of the expansion of Bantu-speaking peoples in Africa***
. Cesar A. Fortes-Lima, Concetta Burgarella, Rickard Hammarén, Anders Eriksson, Mário Vicente, Cecile Jolly, Armando Semo, Hilde Gunnink, Sara Pacchiarotti, Leon Mundeke, Igor Matonda, Joseph Koni Muluwa, Peter Coutros, Terry S. Nyambe, Cirhuza Justin Cikomola, Vinet Coetzee, Minique de Castro, Peter Ebbesen, Joris Delanghe, Mark Stoneking, Larry Barham, Marlize Lombard, Anja Meyer, Maryna Steyn, Helena Malmström, Jorge Rocha, Himla Soodyall, Brigitte Pakendorf, Koen Bostoen, and Carina M. Schlebusch. 2023. doi: [10.1038/s41586-023-06770-6](
https://www.nature.com/articles/s41586-023-06770-6).

###### Acronyms: BSP means “Bantu-speaking populations” and RHG means “African rainforest hunter-gatherers”
###### Of note, some figures were plotted together in the same file (e.g., different projections of the same PCA results). To visualise each figure from some panel you need to click on the taps at the top-corner left.
###### On the right side of the interactive plots there are different tools to scroll, zoom-in and zoom-out in the plot with the mouse (using Box Zoom, Wheel Zoom, Wheel Zoom on the y-axis, Zoom in on the x-axis, Undo, and Reset), or to Save the plots as a figure in high resolution (*.svg or *.png format). 
###### ![alt text](https://github.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/blob/main/03-Other_Methods/tools.png?raw=true)

## Main Figures
##### &emsp;[Figure 1 | Population structure within sub-Saharan African populations](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-Main_Fig.s/Fig_1.html)

##### &emsp;[Figure 2 | Population structure, admixture dates and fractions](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-Main_Fig.s/Fig_2.pdf)

##### &emsp;[Figure 3 | Population structure patterns in BSP on the admixture-masked dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-Main_Fig.s/Fig_3.html)

##### &emsp;[Figure 4 | Patterns of genetic diversity in BSP](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-Main_Fig.s/Fig_4.pdf)

##### &emsp;[Figure 5 | Migration routes and rates in BSP](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-Main_Fig.s/Fig_5.pdf)

## Extended Data Figures and Tables

##### &emsp;[ED Fig. 1 | Geographical locations of modern and ancient populations](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_1.html)

##### &emsp;[ED Fig. 2 | Dimensionality reduction methods applied in this study](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_2.pdf)

##### &emsp;[ED Fig. 3 | Contour maps of each ADMIXTURE result at K=12](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_3.pdf)

##### &emsp;[ED Fig. 4 | Comparisons between EEMS and ADMIXTURE results](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_4.pdf)

##### &emsp;[ED Fig. 5 | Ancestry-specific PCA after removing Himba and Herero populations](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_5.pdf)

##### &emsp;[ED Fig. 6 | Haplotype heterozygosity (HH) and linkage disequilibrium (LD) distribution in BSP](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_6.pdf)

##### &emsp;[ED Fig. 7 | TreeMix for the masked Only-BSP dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_7.pdf)

##### &emsp;[ED Fig. 8 | ADMIXTURE proportions of ancient and modern populations](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_8.pdf)

##### &emsp;[ED Table 1 | Genetic affinity of BSP to non-BSP estimated using f3-statistics](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Table_1.pdf)

##### &emsp;[ED Table 2 | Genetic affinity of ancient individuals to modern BSP](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Table_2.pdf)


## Supplementary Material

### 1- Geographical locations
#
##### &emsp;[Supplementary Fig. 2a | New and previously reported modern-day populations included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_02a_Map.html)

##### &emsp;[Supplementary Fig. 3a | Labels of all the groups included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_03a_Map.html)

##### &emsp;[Supplementary Fig. 3b | Labels of all the populations included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_03b_Map.html)

##### &emsp;[Supplementary Fig. 4a | Labels of all the groups included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_04a_Map.html)

##### &emsp;[Supplementary Fig. 4b | Labels of all the populations included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_04b_Map.html)

##### &emsp;[Supplementary Fig. 4c | Labels of all the groups included in the Only-BSP dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_04c_Map.html)

##### &emsp;[Supplementary Fig. 4d | Labels of all the populations included in the Only-BSP dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_04d_Map.html)


### 2- Dimensionality reduction methods (DRM)
#
##### &emsp;[Supplementary Fig. 6a | UMAP approach on the basis of genotype data of the groups included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_06a_UMAP.html)

##### &emsp;[Supplementary Fig. 6b | UMAP approach on the basis of genotype data of the populations included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_06b_UMAP.html)

##### &emsp;[Supplementary Fig. 7 | PCA plots for all the Bantu-speaking groups included in the Only-BSP dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_07_PCA_Only-BSP_Groups.html)

##### &emsp;[Supplementary Fig. 7 | PCA plots for all the Bantu-speaking populations included in the Only-BSP dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_07_PCA_Only-BSP_Populations.html)

##### &emsp;[Supplementary Fig. 8 | PCA plots for selected sub-Saharan African groups included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_08_PCA_SSA_Groups.html)

##### &emsp;[Supplementary Fig. 8 | PCA plots for selected sub-Saharan African populations included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_08_PCA_SSA_Populations.html)

##### &emsp;[Supplementary Fig. 11 | PCA plots for all the groups included in the AfricanNeo dataset.](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_11_PCA_AfricanNeo_Groups.html)

##### &emsp;[Supplementary Fig. 11 | PCA plots for all the populations included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_11_PCA_AfricanNeo_Populations.html)

##### &emsp;[Supplementary Fig. 12a | PCA-UMAP approach on the basis of genotype data included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_12a_PCA-UMAP.html)

##### &emsp;[Supplementary Fig. 12b | PCA-UMAP approach on the basis of genotype data included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_12b_PCA-UMAP.html)

##### &emsp;[Supplementary Fig. 13a | GCAE approach for all the groups included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_13a_GCAE.html)

##### &emsp;[Supplementary Fig. 13b | GCAE approach for all the populations included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_13b_GCAE.html)

##### &emsp;[Supplementary Fig. 13c | GCAE approach for all the groups included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_13c_GCAE.html)

##### &emsp;[Supplementary Fig. 13d | GCAE approach for all the populations included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_13d_GCAE.html)

### 3- Other methods
#
#### - Genome-wide runs of homozygosity (ROH)

##### &emsp;[Supplementary Fig. 40 | All categories of ROH length for all the populations included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_40_All_ROH.html)

##### &emsp;[Supplementary Fig. 41 | All categories of ROH length only for BSP included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_41_BSP_ROH.html)

#### - Effective population sizes estimated using IBDNe

##### &emsp;[Supplementary Fig. 49a | Estimated effective population sizes (Ne) for selected BSP](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_49a_IBDNe.html)

##### &emsp;[Supplementary Fig. 49b | Shaded areas of Suppl. Fig. 49a between the lower and the upper 95% confidence interval](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_49b_IBDNe.html)

#### - F<sub>ST</sub> map for the masked Only-BSP dataset. 

##### &emsp;[Supplementary Fig. 83a | F<sub>ST</sub> map for all the BSP included in the AfricanNeo datase](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_83a_Fst_Map.html)

##### &emsp;[Supplementary Fig. 83b | F<sub>ST</sub> map for BSP included in the AfricanNeo dataset except for the Lozi population](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_83b_Fst_Map.html)

### 4- Plots of Ancient DNA analyses
#
##### &emsp;[Supplementary Fig. 95 | Geographical locations of aDNA individuals included in this study.](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/04-aDNA/Suppl_Fig_95_aDNA_Map.html)

##### &emsp;[Supplementary Fig. 96_right column | PCA highlighting with colors present-day groups and aDNA individuals.](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/04-aDNA/Suppl_Fig_96_PCA_aDNA_Group_Colour.html)

##### &emsp;[Supplementary Fig. 96_left column | PCA highlighting with colors only aDNA individuals](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/04-aDNA/Suppl_Fig_96_PCA_aDNA_Group_Gray.html)

##### &emsp;[Supplementary Fig. 96 | PCA highlighting with colors present-day populations and aDNA individuals.](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/04-aDNA/Suppl_Fig_96_PCA_aDNA_Population_Colour.html)

##### &emsp;[Supplementary Fig. 97a | PCA-UMAP highlighting with colors only aDNA individuals.](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/04-aDNA/Suppl_Fig_97a_PCA_aDNA.html)

##### &emsp;[Supplementary Fig. 97b | PCA-UMAP highlighting with colors present-day groups and aDNA individuals.](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/04-aDNA/Suppl_Fig_97b_PCA_aDNA.html)

##### &emsp;[Supplementary Fig. 97c | PCA-UMAP highlighting with colors present-day populations and aDNA individuals.](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/04-aDNA/Suppl_Fig_97c_PCA_aDNA.html)
#

### 5- Supplementary Linguistic Data
Cognates are lexical roots originating in one and the same ancestral lexeme which related languages share through inheritance from a common ancestor. Here, we provide updated cognacy judgments (5,355 in total) and a binary-coded root-meaning association matrix using Lexedata. This lexical dataset is available in Cross-Linguistic Data Format (CLDF) here: [cognates.csv, cognatesets.csv, forms.csv, languages.csv, parameters.csv, sources.bib, and Wordlist-metadata.json](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/tree/main/05-Linguistic_Data).

###
### Additional information
- This study was funded by the European Research Council (ERC) under the grant AfricanNeo (“The African Neolithic: A genetic perspective”; grant agreement ID: 759933), granted to Prof. Carina Schlebusch.

- SNP array genotype data of modern-day African populations and whole-genome data of ancient DNA individuals generated in this project were made available through the European Genome-phenome Archive (EGA) data repository.
#
### EGA accessory numbers for data access:

#### AfricanNeo aDNA Study EGA ID: EGAD00001011320. See https://ega-archive.org/datasets/EGAD00001011320
Whole-genome sequencing data from 12 Late Iron Age individuals (6 from Zambia and 6 from South Africa).
Encrypted raw data are available for direct download in *.bam format.
#
#### AfricanNeo ModernDNA Study EGA ID: EGAS50000000006. See https://ega-archive.org/studies/EGAS50000000006
Genome-wide SNP data of 1,763 participants, including 1,526 Bantu speakers from 147 populations across 14 African countries.
Encrypted raw autosomal data are available in *.tped and *.tfam format.

##### - AfricanNeo_A dataset, or Soodyall's Collection, 1027 samples: EGAD50000000008. See https://ega-archive.org/datasets/EGAD50000000008 

##### - AfricanNeo_B dataset, or Pakendorf's Collection, 156 samples: EGAD50000000006. See https://ega-archive.org/datasets/EGAD50000000006 

##### - AfricanNeo_C dataset, or Bostoen's Collection, 300 samples: EGAD50000000009. See https://ega-archive.org/datasets/EGAD50000000009 

##### - AfricanNeo_D dataset, or Delanghe's Collection, 151 samples: EGAD50000000010. See https://ega-archive.org/datasets/EGAD50000000010 

##### - AfricanNeo_E dataset, or Coetzee's Collection, 100 samples: EGAD50000000011. See https://ega-archive.org/datasets/EGAD50000000011 

##### - AfricanNeo_F dataset, or Ebbesen's Collection, 29 samples: EGAD50000000007. See https://ega-archive.org/datasets/EGAD50000000007 

#
#### AfricanNeo Data Access Committee ID: EGAC00001003398. See https://ega-archive.org/dacs/EGAC00001003398 

#### Data Access Policy EGA ID: EGAP00001003469. See https://i-am-an-african.net/documents

#
### Contact person of the AfricanNeo Data Access Committee (DAC):
#### Prof. Carina Schlebusch. Email: carina.schlebusch@ebc.uu.se
Human Evolution Program, Department of Organismal Biology, 
Evolutionary Biology Centre, Uppsala University
Norbyvägen 18C, SE-752 36 Uppsala, Sweden
##
### Contact person of this repository:
#### Cesar Fortes-Lima, PhD. Uppsala University, Sweden. Email: cesar.fortes-lima@ebc.uu.se
More scripts and figures are available upon request.

