# Expansion_of_BSP_peer-reviewed_article
This repository contains online material from the AfricanNeo Project.

###
### - Citation
#
***The genetic legacy of the expansion of Bantu-speaking peoples in Africa***
. Cesar A. Fortes-Lima, Concetta Burgarella, Rickard Hammarén, Anders Eriksson, Mário Vicente, Cecile Jolly, Armando Semo, Hilde Gunnink, Sara Pacchiarotti, Leon Mundeke, Igor Matonda, Joseph Koni Muluwa, Peter Coutros, Terry S. Nyambe, Cirhuza Cikomola, Vinet Coetzee, Minique de Castro, Peter Ebbesen, Joris Delanghe, Mark Stoneking, Larry Barham, Marlize Lombard, Anja Meyer, Maryna Steyn, Helena Malmström, Jorge Rocha, Himla Soodyall, Brigitte Pakendorf, Koen Bostoen & Carina M. Schlebusch. 2023. doi: 10.1101/2023.04.03.535432

Preprint available online at *bioRxiv*: http://biorxiv.org/content/early/2023/04/05/2023.04.03.535432.abstract

BSP = “Bantu-speaking populations” 


# Main Figures

##### [Figure 1 | Population structure within sub-Saharan African populations](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-Main_Fig.s/Fig_1.html)
###### To visualise each figure from this panel you need to click on the taps at the top-corner left.

# Extended Data Figures and Tables

##### [ED Fig. 1 | Geographical locations of modern and ancient populations](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_1.pdf)

##### [ED Fig. 2 | Dimensionality reduction methods applied in this study](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_2.pdf)

##### [ED Fig. 3 | Contour maps of each ADMIXTURE result at K=12](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_3.tif)

##### [ED Fig. 4 | Comparisons between EEMS and ADMIXTURE results](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_4.pdf)

##### [ED Fig. 5 | Ancestry-specific PCA after removing Himba and Herero populations](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_5.pdf)

##### [ED Fig. 6 | Haplotype heterozygosity (HH) and linkage disequilibrium (LD) distribution in BSP](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_6.pdf)

##### [ED Fig. 7 | TreeMix for the masked Only-BSP dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_7.pdf)

##### [ED Fig. 8 | ADMIXTURE proportions of ancient and modern populations](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Fig_8.pdf)

##### [ED  Table 1 | Genetic affinity of BSP to non-BSP estimated using f3-statistics](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Table_1.pdf)

##### [ED Table 2 | Genetic affinity of ancient individuals to modern BSP](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/00-ExtData/ED_Table_2.pdf)


# Supplementary Material

## 1- Geographical locations

##### [Supplementary Fig. 2a | Labels of all the groups included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_02a_Map.html)

##### [Supplementary Fig. 3a | Labels of all the groups included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_03a_Map.html)

##### [Supplementary Fig. 3b | Labels of all the populations included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_03b_Map.html)

##### [Supplementary Fig. 4a | Labels of all the groups included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_04a_Map.html)

##### [Supplementary Fig. 4b | Labels of all the populations included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_04b_Map.html)

##### [Supplementary Fig. 4c | Labels of all the groups included in the Only-BSP dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_04c_Map.html)

##### [Supplementary Fig. 4d | Labels of all the populations included in the Only-BSP dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/01-Maps/Suppl_Fig_04d_Map.html)


## 2- Dimensionality reduction methods (DRM)

##### [Supplementary Fig. 6a | UMAP approach on the basis of genotype data included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_06a_UMAP.html)

##### [Supplementary Fig. 6b | UMAP approach on the basis of genotype data included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_06b_UMAP.html)

##### [Supplementary Fig. 7 | PCA plots for all the Bantu-speaking groups included in the Only-BSP dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_07_PCA_Only-BSP_Groups.html)

##### [Supplementary Fig. 7 | PCA plots for all the Bantu-speaking populations included in the Only-BSP dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_07_PCA_Only-BSP_Populations.html)

##### [Supplementary Fig. 8 | PCA plots for selected sub-Saharan African groups included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_08_PCA_SSA_Groups.html)

##### [Supplementary Fig. 8 | PCA plots for selected sub-Saharan African populations included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_08_PCA_SSA_Populations.html)

##### [Supplementary Fig. 11 | PCA plots for all the groups included in the AfricanNeo dataset.](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_11_PCA_AfricanNeo_Groups.html)

##### [Supplementary Fig. 11 | PCA plots for all the populations included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_11_PCA_AfricanNeo_Populations.html)

##### [Supplementary Fig. 12a | PCA-UMAP approach on the basis of genotype data included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_12a_PCA-UMAP.html)

##### [Supplementary Fig. 12b | PCA-UMAP approach on the basis of genotype data included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_12b_PCA-UMAP.html)

##### [Supplementary Fig. 13a | GCAE approach for all the groups included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_13a_GCAE.html)

##### [Supplementary Fig. 13b | GCAE approach for all the populations included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_13b_GCAE.html)

##### [Supplementary Fig. 13c | GCAE approach for all the groups included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_13c_GCAE.html)

##### [Supplementary Fig. 13d | GCAE approach for all the populations included in the Only-African dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/02-DRM/Suppl_Fig_13d_GCAE.html)

## 3- Other methods

### - Genome-wide runs of homozygosity (ROH)

##### [Supplementary Fig. 40 | All categories of ROH length for all the populations included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_40_All_ROH.html)

##### [Supplementary Fig. 41 | All categories of ROH length only for BSP included in the AfricanNeo dataset](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_41_BSP_ROH.html)

### - Effective population sizes estimated using IBDNe

##### [Supplementary Fig. 49a | Estimated effective population sizes (Ne) for selected BSP](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_49a_IBDNe.html)

##### [Supplementary Fig. 49b | Shaded areas of Suppl. Fig. 49a between the lower and the upper 95% confidence interval](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_49b_IBDNe.html)

### - FST map for the masked Only-BSP dataset. 

##### [Supplementary Fig. 83a | Fst map for all the BSP included in the AfricanNeo datase](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_83a_Fst_Map.html)

##### [Supplementary Fig. 83b | Fst map for BSP included in the AfricanNeo dataset except for the Lozi population](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/main/03-Other_Methods/Suppl_Fig_83b_Fst_Map.html)
#

## 4- Ancient DNA
### -

## 5- Supplementary Linguistic Data
Cognates are lexical roots originating in one and the same ancestral lexeme which related languages share through inheritance from a common ancestor. Here, we provide updated cognacy judgments (5,355 in total) and a binary-coded root-meaning association matrix using Lexedata. This lexical dataset is available in Cross-Linguistic Data Format (CLDF; files: [cognates.csv, cognatesets.csv, forms.csv, languages.csv, parameters.csv, sources.bib, and Wordlist-metadata.json](https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_peer-reviewed_article/tree/main/05-Linguistic_Data)).

#
###
# Additional information

The AfricanNeo Project (ERC grant agreement ID: 759933) was led by Carina Schlebusch.

### AfricanNeo aDNA Study EGA ID: EGAS00001007519
Whole-genome sequencing data from 12 Late Iron Age individuals
###### - EGA AfricanNeo_aDNA Dataset ID: EGAD00001011320
Encrypted raw data are available in *.bam format.

### AfricanNeo ModernDNA Study EGA ID: EGAS00001007515
Genome-wide SNP data of 1,763 participants, including 1,526 Bantu speakers from 147 populations across 14 African countries.
Encrypted raw autosomal data are available in *.tped and *.tfam format.
#
###### - EGA AfricanNeo_A Dataset ID: EGAD0000
#
###### - EGA AfricanNeo_B Dataset ID: EGAD0000
#
###### - EGA AfricanNeo_C Dataset ID: EGAD0000
#
###### - EGA AfricanNeo_D Dataset ID: EGAD0000
#
###### - EGA AfricanNeo_E Dataset ID: EGAD0000
#
###### - EGA AfricanNeo_F Dataset ID: EGAD0000
#

### Data Access Committee EGA ID: EGAC00001003398
Short name: AfricanNeo Data Access Committee

### Data Access Policy EGA ID: EGAP00001003469
Policy URL: https://i-am-an-african.net/documents

### Contact person of the AfricanNeo Data Access Committee (DAC):
#### Carina Schlebusch. Email: carina.schlebusch@ebc.uu.se
Associate Professor / Senior Lecturer. Human Evolution Program
Department of Organismal Biology, Evolutionary Biology Centre, Uppsala University

Norbyvägen 18C, SE-752 36 Uppsala, Sweden


##
